### PR TITLE
[SMALLFIX] [branch-1.4] Fix ufs capability test with object storage

### PR DESCRIPTION
**Pull request by ifcharming**
It's expected to get invalid ufs capability for object storage.

### CODE REVIEW

**Comment by ifcharming** [comment]
@madanadit @jsimsa

File: tests/src/test/java/alluxio/master/AlluxioMasterRestApiTest.java [hunk 1]
```diff
@@ -167,7 +168,12 @@ public void getUptimeMs() throws Exception {
   @Test
   public void getUfsCapacity() throws Exception {
     Capacity ufsCapacity = getInfo(NO_PARAMS).getUfsCapacity();
-    Assert.assertTrue(ufsCapacity.getTotal() > 0);
+    if (CommonUtils.isUfsObjectStorage(mFileSystemMaster.getUfsAddress())) {
+      // Object storage ufs capability is always invalid.
```
Comment (calvinjia) [inline tests/src/test/java/alluxio/master/AlluxioMasterRestApiTest.java]: `Capacity`?

Comment (ifcharming) [inline tests/src/test/java/alluxio/master/AlluxioMasterRestApiTest.java]: oops, done.

**Comment by madanadit** [comment]
LGTM

**Pull request merged**